java/client/src/org/openqa/selenium/remote/RemoteWebDriver.java

```diff
@@ -322,6 +325,14 @@ public String getCurrentUrl() {
     }
   }
 
+  @Override
+  public Pdf print(Object printOptions) throws WebDriverException {
```

Comment:
You do a cast immediately below to `PrintOptions`, and so you've made a (fast, easy to fix) compile-time error into a (found later in the development cycle, harder to identify and therefore fix) runtime error.

`Pdf print(PrintOptions)` please.